config/functions/bootstrap.js

```diff
@@ -0,0 +1,48 @@
+module.exports = async () => {
+  // Check if the plugin users-permissions is installed because the comments needs it
+  if (Object.keys(strapi.plugins).indexOf('users-permissions') === -1) {
+    throw new Error(
+      'In order to make the comments plugin works the users-permissions one is required'
```

Comment:
`works` -> `work`
`the users-permissions one` -> `the users-permissions plugin`